.java-version

```diff
@@ -0,0 +1 @@
+oracle64-10.0.2
```

Comment:
shouldn't we be using OpenJDK like in our Dockerimages?

Reply:
This file shouldn't even be there. I'll remove it